### PR TITLE
New sanity test for build (Unmockable)

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3507)

## Description
Changing the Sanity Test that fails on quota error in many builds with a new one.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Tests/scripts/collect_tests_and_content_packs.py [hunk 1]
```diff
@@ -31,7 +31,8 @@
     'Sanity Test - Playbook with integration',
     'Sanity Test - Playbook with no integration',
     'Sanity Test - Playbook with mocked integration',
-    'Sanity Test - Playbook with Unmockable Integration',
+    'Sanity Test - Playbook with Unmockable Integration check'
```
Comment (dorschw) [inline Tests/scripts/collect_tests_and_content_packs.py]: I'd suggest removing the check, maybe adding `whois` (btw, is it unmockable?) to make it clearer

File: Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py [hunk 2]
```diff
@@ -1091,12 +1091,12 @@ def test_pack_ignore_test_is_skipped(mocker):
                 id_set=fake_id_set
             )
 
-            assert content_packs == {"HelloWorld", "Gmail", "Base", "DeveloperTools", pack_name}
+            assert content_packs == {"Whois", "HelloWorld", "Gmail", "Base", "DeveloperTools", pack_name}
```
Comment (yuvalbenshalom) [inline Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py:1094]: do we still need Gmail? need to check the other tests.

File: Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py [hunk 3]
```diff
@@ -615,7 +615,7 @@ def test_no_change(self, mocker):
         get_modified_files_ret = create_get_modified_files_ret()
         filterd_tests, content_packs = get_mock_test_list(get_modified_files_ret, mocker)
 
-        assert content_packs == {"Gmail", "HelloWorld", "Base", "DeveloperTools"}
+        assert content_packs == {"Gmail", "HelloWorld", "Base", "DeveloperTools", "Whois"}
```
Comment (yuvalbenshalom) [inline Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py:618]: do we still need Gmail? need to check the other tests.

File: Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py [hunk 4]
```diff
@@ -590,7 +590,7 @@ def test_all_tests(self):
         assert 'TestCommonPython' in filterd_tests
         # verify all sanity tests are collected
         assert len(SANITY_TESTS - filterd_tests) == 0
-        assert content_packs == {'DeveloperTools', 'Base', 'HelloWorld', 'Gmail'}
+        assert content_packs == {'Whois', 'DeveloperTools', 'Base', 'HelloWorld', 'Gmail'}
```
Comment (yuvalbenshalom) [inline Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py:593]: do we still need Gmail? need to check the other tests.

File: Tests/scripts/collect_tests_and_content_packs.py [hunk 5]
```diff
@@ -1303,9 +1303,9 @@ def get_test_list_and_content_packs_to_install(files_string,
             tests.update(SANITY_TESTS)
             logging.debug("Adding HelloWorld to tests as most of the sanity tests requires it.")
             logging.debug(
-                "Adding Gmail to packs to install as 'Sanity Test - Playbook with Unmockable Integration' uses it"
+                "Adding Whois to packs to install as 'Sanity Test - Playbook with Unmockable Whois Integration' uses it"
             )
-            packs_to_install.update(["HelloWorld", "Gmail"])
+            packs_to_install.update(["HelloWorld", "Gmail", "Whois"])
```
Comment (yuvalbenshalom) [inline Tests/scripts/collect_tests_and_content_packs.py:1308]: do we still need Gmail? need to check the other tests.

**Review by yuvalbenshalom — COMMENTED** [review]
just need to check if we still need to include gmail in the installed packs.
depending on the other sanity playbooks.

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**